application/app/controllers/upload_files_controller.rb

```diff
@@ -106,19 +105,20 @@ def list_files(path, limit: 100)
     if File.file?(path)
       return [OpenStruct.new(
         fullpath: File.expand_path(path),
-        filename: File.basename(path),
+        filename: File.join('/', File.basename(path)),
```

Comment:
I think we should use something like `File::SEPARATOR` for cross-platform compatibility

Reply:
ok